src/ebay-filter-menu/filter-menu.tsx

```diff
@@ -208,4 +255,4 @@ const EbayFilterMenu: FC<EbayFilterMenuProps> = ({
     )
 }
 
-export default EbayFilterMenu
+export default withForwardRef(EbayFilterMenu)
```

Comment:
are we not on react 19? should we just use this as a PROP?

Reply:
No [we do support](https://github.com/eBay/ebayui-core-react/blob/main/package.json#L57-L58) React 16, we will when we upgrade the apps and will release a breaking change